gapic/src/main/com/google/gapid/perfetto/views/CpuPanel.java

```diff
@@ -159,6 +168,14 @@ private void renderSlices(RenderContext ctx, CpuTrack.Data data, double h) {
       }
     }
 
+    // Draw bounding rectangles after all the slices are rendered, so that the border is on the top.
+    ctx.setForegroundColor(SWT.COLOR_BLACK);
```

Comment:
make the color a style constant. Black doesn't really work in dark mode.

Reply:
Will change the color for the bounding boxes in a new PR following.